rideshare.rb

```diff
@@ -0,0 +1,141 @@
+all_rides = [
+  {
+    "driver id" => "DR0001",
+    "rides" => [
+      {
+        "date" => "3rd Feb 2016",
```

Comment:
In general, in Ruby it is considered good style to use symbols as the keys in hashes.